CFC_WebApp/api/cfc_webapp.py

```diff
@@ -282,6 +281,26 @@ def getCarbonCompare():
     logging.debug("No overriding client result for user %s, returning default" % user_uuid)
   return default.getResult(user_uuid)
 
+@get('/compare')
```

Comment:
Have you actually tested this with both @get and @post endpoints and ensured that it works? Can you update the testing done section with detailed tests.